src/index.html

```diff
@@ -3,12 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body> 
+    <header class="header"> 
+      <div class="logo"> 
+        <a href="/" class="logo-link"> 
+          <img src="./images/logo.png" alt="logo"> 
+        </a> 
+      </div>
+
+      <nav class="nav"> 
+        <ul class="nav_list"> 
+          <li class="nav_item"> 
+            <a href="#Apple" class="is-active nav_link">Apple</a>
+          </li> 
+          <li class="nav_item"> 
+            <a href="#Samsung" class="nav_link">Samsung</a>
+          </li> 
+          <li class="nav_item"> 
+            <a href="#Smartphones" class="nav_link">Smartphones</a>
+          </li> 
+          <li class="nav_item"> 
+            <a href="Laptops" class="nav_link" 
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a 
              href="Laptops" 
              class="nav_link" 
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Reply:
Fix the code style, please 
![image](https://github.com/mate-academy/layout_moyo-header/assets/99403198/210b6f84-c546-464a-b28a-9d748cc020cd)